### DISCUSSION TITLE
Increasing pods per node limit

### DISCUSSION THREAD

**Discussion by serhiynovos** [rancher/rke2#6185] [Q&A] [answered]
Hi. I have RKE2 cluster provisioned by Rancher with Cilium CNI. I have 3 high capacity host servers and I want to use this cluster to run a lot of client deployments. One issue what I'm facing that Kubernetes by default has 110 pods per node and I know it somehow related to ip tables but if we use cilium we can use more pods per node. 
Currently I have situation that I'm reaching my PODs limit that CPU or Memory limit. 
<img width="1407" alt="image" src="https://github.com/rancher/rke2/assets/9201885/169c0afd-f133-4c26-bd06-35743cbdda2f">

For deal with it I see two possible scenarios:
1. Run more smaller Worker VMs to on the same hardware I'll be able to run more pods.
2. Increase Max Pod limits.

Can you please helm me to understand what approach will be better in my scenario ?

Thanks

**Comment by dipaladu369** [comment]
Have you been able to solve the issue.

**Reply by dlc-letelier** [reply, marked as answer]
Look here:

https://github.com/rancher/rke2/issues/4251#issuecomment-1552537970
